lib/Settings/Personal/Section.php

```diff
@@ -32,29 +32,23 @@
 use OCP\Settings\IIconSection;
 
 class Section implements IIconSection {
```

Comment:
Wait, is that no problem that this section is defined twice then?

Reply:
Nope, not a problem to have multiple registrations with the same id ;)